rpms/README.md

```diff
@@ -24,30 +24,34 @@ To build an RPM package, you need to download this repository and use the `gener
 
     Usage: ./generate_rpm_package.sh [OPTIONS]
 
-        -b, --branch <branch>     [Required] Select Git branch or tag e.g. master
-        -t, --target <target>     [Required] Target package to build [manager/api/agent].
-        -a, --architecture <arch> [Optional] Target architecture of the package [x86_64/i386].
-        -r, --revision <rev>      [Optional] Package revision that append to version e.g. x.x.x-rev
-        -l, --legacy              [Optional] Build package for CentOS 5.
-        -s, --store <path>        [Optional] Set the destination path of package.
-        -j, --jobs <number>       [Optional] Number of parallel jobs when compiling.
-        -p, --path <path>         [Optional] Installation path for the package. By default: /var.
-        -d, --debug               [Optional] Build the binaries with debug symbols and create debuginfo packages. By default: no.
-        -c, --checksum <path>     [Optional] Generate checksum on the desired path (by default, if no path is specified it will be generated on the same directory than the package).
-        -h, --help                Show this help.
+        -b,   --branch <branch>     [Required] Select Git branch or tag e.g. master
+        -t,   --target <target>     [Required] Target package to build [manager/api/agent].
+        -a,   --architecture <arch> [Optional] Target architecture of the package [x86_64/i386].
+        -r,   --revision <rev>      [Optional] Package revision that append to version e.g. x.x.x-rev
+        -l,   --legacy              [Optional] Build package for CentOS 5.
+        -s,   --store <path>        [Optional] Set the destination path of package. By default, an output folder will be created.
+        -j,   --jobs <number>       [Optional] Number of parallel jobs when compiling.
+        -p,   --path <path>         [Optional] Installation path for the package. By default: /var.
+        -d,   --debug               [Optional] Build the binaries with debug symbols and create debuginfo packages. By default: no.
+        -c,   --checksum <path>     [Optional] Generate checksum on the desired path (by default, if no path is specified it will be generated on the same directory than the package).
+        -src, --source              [Optional] Generate the source package in the destination directory
```

Comment:
This option should be `--src` only.

Reply:
Done in https://github.com/wazuh/wazuh-packages/pull/322/commits/1c0fc432f8a6a01bc925799249c0761244cb4f1f

Reply:
I think is better `--src` than `-src`.